controllers/comments.controller.js

```diff
@@ -0,0 +1,18 @@
+const { selectArticle } = require("../models/articles.model");
+const { selectCommentsByArticleId } = require("../models/comments.model");
+
+exports.getCommentsByArticleId = (request, response, next) => {
+    const articleId = request.params.article_id;
+    const promises = [
+        selectArticle(articleId),
+        selectCommentsByArticleId(articleId)
+    ];
+    return Promise.all(promises).then((resolved) => {
+        const [ articleResult, commentsResult ] = resolved;
+        if (articleResult.rowCount === 0) {
+            return Promise.reject({ status: 404, msg: 'not found' });
+        }
```

Comment:
Would be neater to have the model do the promise rejecting. That way if the article doesn't exist, it will fall straight into the catch block of your controller after the promise.all 🙂 You don't have to change this, just an observation